src/js/main.js

```diff
@@ -344,6 +345,35 @@ class Site {
       }
     });
   }
+
+  handleMouseup() {
+    $('body')
+    .off('mousemove', this.handleDragging)
+    .off('mouseup', this.handleMouseup);
+  }
+
+  handleDragging(e) {
+    const left = window.drag.offset0.left + (e.pageX - window.drag.pageX0);
+    const top = window.drag.offset0.top + (e.pageY - window.drag.pageY0);
+    $(window.drag.elem)
+    .offset({top: top, left: left});
+  }
+
+  bindPostitDrag(e) {
+    let _this = this;
+
+    $('#postit').mousedown(function(e){
+      window.drag = {};
```

Comment:
since were saving `Site` as `_this`, we can put the drag object there instead of on `window` which is preferred. Then, in `handleDragging()` you can call it by `this.drag`